nix/haskell-extra.nix

```diff
@@ -6,6 +6,38 @@
 ############################################################################
 { pkgs, index-state, checkMaterialization }:
 {
+  Agda = pkgs.haskell-nix.hackage-package {
+    name = "Agda";
+    version = "2.6.1";
+    plan-sha256 = "1l8fviiy9baqys6gq1s1lrpczv355yl0w4p30ma2n9y71xv356pq";
+    inherit index-state checkMaterialization;
+    modules = [{
+      # Agda is a huge pain. They have a special custom setup that compiles the interface files for
```

Comment:
Ah, I think I get it now. So, this problem is related to some minimal library that is builtin to Agda (I am guessing `Agda.Builtin.*` etc) rather than the 'standard' library which is entirely separate?

Reply:
Yep.